### PR TITLE
Globus Tutorial  For Transferring Data 

### PR DESCRIPTION
**Pull request by Premas**
# Pull Request
Globus Tutorial  For Transferring Data 

## Overview
The following section of the Globus Tutorial covers transferring data between LTS and Cheaha, as well as moving shared collections to project directories on these platforms.

## Proposed Changes
Added the following section. 
1. How to transfer data from a shared collection to my project directory on Cheaha?
2. How to transfer data from a shared collection to my group's LTS allocation?
3. How to transfer data between LTS and Cheaha?

## Related Issues
Fixes part of #800

### CODE REVIEW

File: docs/data_management/tutorial/globus_individual_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+To transfer data from a shared collection to a project directory on Cheaha, you may follow these steps:
+
+1. Select the Shared Collection:  Click on the Collection tab, then go to the "Shared With You" section and choose the appropriate collection name, for instance, "test collection cheaha". Once selected, you will see the data shared with you.
+
+1. On the other side of the Collection tab, select the UAB Cheaha collection. Authenticate your account if prompted, and navigate to your project directory, e.g., `/data/project/xxx`, where you want to transfer the data.
+
+1. Start the Transfer: Click the "Start" button on the shared collection side. The data, such as the folder `test`, will then be transferred to your specified project directory.
+
+<!--
```
Comment (wwarriner) [inline docs/data_management/tutorial/globus_individual_tutorial.md:13]: Great use of HTML comments!

**Review by wwarriner — COMMENTED** [review]
Looks good so far!

**Comment by Premas** [comment]
Added images to this box folder, https://uab.app.box.com/folder/282177791476.

**Comment by wwarriner** [comment]
Converted to draft. This will be integrated into #829

**Pull request merged**